ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/api/CommitMetadata.scala

```diff
@@ -14,12 +20,36 @@ sealed trait CommitMetadata {
     *         submissions
     */
   def estimatedInterpretationCost: Option[Long]
+
+  def inputKeys(serializationStrategy: StateKeySerializationStrategy): Iterable[Bytes]
```

Comment:
Do we need to be java-friendly in this API?

Reply:
No, not here. I.e., we may want to support a Java-friendly API in parallel to the Scala APIs in the future.